packages/elastic-apm-synthtrace/src/lib/utils/logger.ts

```diff
@@ -1,67 +0,0 @@
-/*
```

Comment:
Was this supposed to be deleted? Looks like `clean_write_targets.ts` still imports it.

Reply:
I had duplicated files, renaming that import to create_logger.ts should work. And yeah you can safely ignore the bootstrap reference errors (I've disabled those in my local setup).

Reply:
Gotcha. How do you disable those? For now I've just ignored them.